sema/harvest/config_build.py

```
@@ -325,8 +326,9 @@ def _assert_valid_sparql_syntax(self, sparql_query):
     def _files_folder(self):
         return [
             f.name
-            for f in Path(self.config_files_folder).rglob("*")
-            if f.is_file() and f.name.endswith((".yml", ".yaml"))
+            for f in getMatchingGlobPaths(
+                includes=["*.yml", "*.yaml"], onlyFiles=True
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)